changelog/trap_memory_error.dd

```diff
@@ -0,0 +1,38 @@
+Page protection error handling can now be registered via `--DRT-memoryError=1`
+
+In environments where attaching a debugger or retrieving a core dump isn't possible or hard,
+on Linux x86_64 one has always been able to attach druntime's  memory handler:
```

Comment:
The code in the `etc.linux.memoryerror` module is defined for both 32 bit and 64 bit.